### ISSUE TITLE
Request - change desktop wallpaper every x time

### ISSUE THREAD

**Issue by dferrandizmont** [gaurav-nelson/unsplash-wallpaper#3]
Hi! First of all thanks for the app, works perfect, but I'd like to know if there is any change to add an option to change it every 1 - 2 -3 - N hours automatically.

Thanks!

**Comment by gaurav-nelson** [comment]
Thank you @dferrandizmont 
That would probably require running the app in background and making that setting. This was more of a weekend project I did in my free time. So I'll have a go at it, but I'm not making any promises.

**Comment by dferrandizmont** [comment]
Nah, sure, just if you have time and want it :). Anyway, thanks for sharing it!
